src/test/java/seedu/address/model/leave/LeaveTest.java

```diff
@@ -124,7 +124,7 @@ public void isSameLeaveMethod() {
                 .withDescription("Alice's Maternity Leave 2 Description").build()));
 
         // different status -> return true
-        assertTrue(ALICE_LEAVE.isSameLeave(new LeaveBuilder(ALICE_LEAVE).withStatus("Approved").build()));
+        assertTrue(ALICE_LEAVE.isSameLeave(new LeaveBuilder(ALICE_LEAVE).withStatus("APPROVED").build()));
```

Comment:
possible to make withStatus take in APPROVED enum instead?

Reply:
The problem here is that the Status constructor (using the enum) is private. We are currently using the factory method for Status at the moment. Let's discuss more abt this in the chat.